infrastrucutre/eks.tf

```diff
@@ -0,0 +1,70 @@
+data "http" "myip" {
```

Comment:
Don't use this as office IP is supposed to be static and if you run this code from other location it will change IP.